codegenerator/cli/src/fuel/abi.rs

```diff
@@ -211,7 +211,7 @@ impl FuelAbi {
                         "b256" | "address" => String.to_ok_expr(),
                         "str" => String.to_ok_expr(),
                         type_field if type_field.starts_with("str[") => String.to_ok_expr(),
-                        "struct Vec" => Array(Box::new(GenericParam(
+                        "struct std::vec::Vec" => Array(Box::new(GenericParam(
```

Comment:
Is this a new abi repr of a standard vec?

Reply:
Yep

Reply:
I think it's the same, but they started prefixing struct names with modules